ratelimit/utils.py

```diff
@@ -6,6 +6,10 @@
 
 from django.conf import settings
 from django.core.cache import caches
+try:
+    from django_redis.cache import RedisCache
+except ImportErro:
```

Comment:
This isn't tested - typo in `ImportError`

Reply:
sorry i select the wrong branch 😞 the correct one was my branch, this was the branch test for the version 2.0.0 that i have fixed in local but not on github :(